src/patternfly/components/Table/templates/table--toggle-all.hbs

```diff
@@ -0,0 +1 @@
+{{#if table-tr--expanded}}{{#> table-td table-tr--expanded="true" table-td--toggle="true" table-td--button--attribute=(concat 'id="' table--id '-expandable-toggle-' table-tr--index '" aria-label="Collapse all" ')}}{{/table-td}}{{else}}{{#> table-td table-td--toggle="true" table-td--button--attribute=(concat 'id="' table--id '-expandable-toggle-' table-tr--index '" aria-label="Expand all" ')}}{{/table-td}}{{/if}}
```

Comment:
Does this need to be on one line or can we make it multi-line so it's easier to read?

Reply:
It was making an extra line break unless I put it all on one line for some reason. I'd personally prefer readable code and live with an extra line break but I figured I'd get called out on it 😝

Reply:
Yah as long as it doesn't break the HTML or make the example code look bad - like a line break or extra space in the log viewer's text in a `<pre>` tag causes the text in the example to look bad) - I'm in favor of readable code.